release-tools/scripts/docker_download_plugins.sh

```diff
@@ -5,6 +5,8 @@
 # Requires yq v4.0.0+
 
 set -e
+CATEGORY=$1; echo CATEGORY $CATEGORY; if [ -z "$CATEGORY" ]; then echo "Please enter plugin category as parameter: elasticsearch/kibana"; exit 1; fi
+ARCHTECTURE=$2; echo ARCHITECTURE $ARCHITECTURE
```

Comment:
While running this script in [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/561/files#diff-ad6f24d7ee0e1d39067adb1ccfffd64f8cde0ba89328dd2b329b1afd78c240d7R36) where have we mentioned the architecture?

Reply:
Make changes to the logic a bit.